Documentation/howto/ovs-with-p4-executables.rst

```diff
@@ -661,6 +661,33 @@ previously configured CONFIG params.
     tdi-portin-id: Port ID for Pipeline in Input Direction
     tdi-portin-id: Port ID for Pipeline in Output Direction
 
+10) Get atrributes for port statistics retrieval::
+
+    $ gnmi-cli get PARAMS
+    $ Example for LINK ports :
+    gnmi-cli get "device:physical-device,name:PORT0,counters" | grep "name\|uint_val" | grep -v "interface\|key\|config\|counters"
+    $ Example for vhost ports :
+    gnmi-cli get "device:virtual-device,name:net_vhost0,counters" | grep "name\|uint_val" | grep -v "interface\|key\|config\|counters"
+    $ Example for TAP ports :
+    gnmi-cli get "device:virtual-device,name:TAP0,counters" | grep "name\|uint_val" | grep -v "interface\|key\|config\|counters"
+
+  .. note::
+
+    Port stats can be retrieved for the ports that are configured through GNMI CLI.
```

Comment:
created through GNMI CLI? And can we also add " PORT0, net_vhost0 and TAP0 corresponds to the name used when creating the ports using GNMI CLI"?

Reply:
Done